cdc/sink/mq/manager/kafka_manager.go

```diff
@@ -112,9 +131,80 @@ func (m *kafkaTopicManager) tryUpdatePartitionsAndLogging(topic string, partitio
 	}
 }
 
-// CreateTopic creates a topic with the given name
-// and returns the number of partitions.
-func (m *kafkaTopicManager) CreateTopic(topicName string) (int32, error) {
+func (m *kafkaTopicManager) getMetadataOfTopics() ([]*sarama.TopicMetadata, error) {
+	var topicList []string
+
+	start := time.Now()
+	m.topics.Range(func(key, value any) bool {
+		topic := key.(string)
+		topicList = append(topicList, topic)
+
+		return true
+	})
+
```

Comment:
Shouldn't Start be in this place?

Reply:
Yes, it should be placed right before `admin.DescribeTopics()`. I'll fix it later.